RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/SiPixelRawToClusterKernel.dev.cc

```diff
@@ -432,7 +432,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     struct FillHitsModuleStart {
       template <typename TAcc>
       ALPAKA_FN_ACC void operator()(const TAcc &acc, SiPixelClustersSoAView clus_view) const {
-        ALPAKA_ASSERT_ACC(TrackerTraits::numberOfModules < 2048);  // easy to extend at least till 32*1024
+        ALPAKA_ASSERT_ACC(TrackerTraits::numberOfModules < 4096);  // easy to extend at least till 32*1024
```

Comment:
where do `2048` or `4096` come from ?

Reply:
For the following block prefix scan to [count module starts](https://github.com/cms-sw/cmssw/blob/149acad808a40f21ff653e884fbe2b2c6d2784d0/RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/SiPixelRawToClusterKernel.dev.cc#L457-L471).

Reply:
Mhm, the limits for the blockPrefixScan are 2048 for Phase1 and 4096 for Phase2.
Can you do something similar for the assertion ?
```suggestion
        if constexpr (isPhase2) {
          // easy to extend at least till 32*1024
          ALPAKA_ASSERT_ACC(TrackerTraits::numberOfModules < 4096);
        } else {
          // for Phase 1 there are only 1856 modules
          ALPAKA_ASSERT_ACC(TrackerTraits::numberOfModules < 2048);
        }
```